service/storage_service.cc

```diff
@@ -127,12 +127,24 @@ session_manager& get_topology_session_manager() {
 }
```

Comment:
Re last commit message: Closes -> Fixes. (Apparently our automation is based on Fixes, not Closes.)

Reply:
changed to `Fixes`